### ISSUE TITLE
S3 provider always uses us-east-1

### ISSUE THREAD

**Issue by ooswald** [cloudquery/cloudquery#8]
I am not sure wether this change was correct:
https://github.com/cloudquery/cloudquery/commit/c6cddaefa5909e466de83ce7fdf3c1c9ec3f1952#diff-ec36749d97d730e19702baa9729041e766dd4134df981ae67702657c0abd515b

Because in 0.4.2 I get this when fetching s3 buckets:
```
2020/12/17 00:04:54 AuthorizationHeaderMalformed: The authorization header is malformed; 
the region 'us-east-1' is wrong; expecting 'eu-central-1'
```

**Comment by korsune** [comment]
@ooswald thanks for bringing this up. I think it is fixed now in [v0.4.3](https://github.com/cloudquery/cloudquery/releases/tag/v0.4.3) , I wasn't able to reproduce the exact same error message, can you please try and re-run this to see if the error is solved?

**Comment by ooswald** [comment]
Awesome, I am happy to confirm this S3 issue is resolved - also the other commit which resolved the access key panic.
